quipucords/api/deployments_report/util.py

```diff
@@ -83,7 +82,7 @@ def create_deployments_csv(deployments_report_dict, request):
     deployment_report_buffer = StringIO()
     csv_writer = csv.writer(deployment_report_buffer, delimiter=',')
 
-    systems_list = deployments_report_dict.get('system_fingerprints')
+    systems_list = deployments_report_dict.get("system_fingerprints")
```

Comment:
why did you change single quotation marks for double on this specific example?

Reply:
this was black recommendation. I should probably have splited this in two separate commits.

Reply:
and this line shouldn't be here. ouch. I moved it to line 98 :facepalm: